<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Other Colors</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_Red">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalSlider_Red">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_Red">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Green">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_Green">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_Green">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Blue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider_Blue">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_Blue">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>graphicsView</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
